tests/e2e/configuration-reload/README.md

```diff
@@ -0,0 +1,172 @@
+# Configuration Reload Test Suite
+
+## Purpose
+This test suite verifies that the Odigos collector correctly reloads its configuration when actions are created or modified, ensuring that observability pipeline changes are applied dynamically without service restarts.
+
+## What is Tested
+
+### Configuration Management
+- **Action creation** - Adding new observability actions to the system
+- **Configuration generation** - Automatic collector configuration updates
+- **Configuration reload** - Hot-reloading collector configuration without restart
+- **Reload verification** - Confirming configuration changes are applied
```

Comment:
I think simply checking the logs does not mean that it's applied. we should check the generated telemetry for that